Library/ViewModels/ProjectPageViewModel.swift

```diff
@@ -198,12 +217,42 @@ public final class ProjectPageViewModel: ProjectPageViewModelType, ProjectPageVi
       .skipNil()
       .observeValues { AppEnvironment.current.cookieStorage.setCookie($0) }
 
+    self.presentMessageDialog = project
+      .takeWhen(self.askAQuestionCellTappedProperty.signal)
+
+    self.showHelpWebViewController = self.projectEnvironmentalCommitmentDisclaimerCellDidTapURLProperty.signal
+      .skipNil()
+      .map(HelpType.helpType)
+      .skipNil()
+
     // We skip the first one here because on `viewDidLoad` we are setting .overview so we don't need a useless emission here
-    self.navigatePageViewController = self.projectNavigationSelectorViewDidSelectProperty.signal
+    self.updateDataSource = self.projectNavigationSelectorViewDidSelectProperty.signal
       .skipNil()
       .map { index in NavigationSection(rawValue: index) }
       .skipNil()
+      .combineLatest(with: project.map(\.extendedProjectProperties).skipNil())
+      .map { navSection, projectProperties in
+        let initialIsExpandedArray = Array(repeating: false, count: projectProperties.faqs.count)
```

Comment:
Yeah right now we are not preserving state as it wasn't explicitly listed in the acceptance criteria. I'm going to look into how much effort it will take to do that after I push the first set of changes up.